### ISSUE TITLE
No field to repeat password for confirmation

### ISSUE THREAD

**Issue by markbourne** [keepassxreboot/keepassxc#5119]
KeePassXC 2.6.0 on Linux Mint 18 with Mate desktop

When editing an entry, there are icons in the password field for "Passwords match so far" and "Passwords do not match", but there is only one password field so nothing to match or not match. The same occurs in the password generator. Tools > Settings > Security > "Require password repeat when it is visible" is set (ticked), although there's no field to repeat the password even if the password is masked.

It looks like the intention is that there should still be a second password field (I'm sure there was in previous versions) - but it doesn't seem to be shown, and entries can be created and the password changed without ever entering it a second time to confirm.

Debug info from Help > About:
```
KeePassXC - Version 2.6.0
Revision: 0765954

Qt 5.5.1
Debugging mode is disabled.

Operating system: Linux Mint 18.3
CPU architecture: x86_64
Kernel: linux 4.15.0-107-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
 libgcrypt 1.8.1
```

**Comment by phoerious** [comment]
There is only a password repeat field for the master key, not for entries. You can always look up entries in plain text, so there is no need for such a field. Any traces left of a password repeat field come from previous versions and should probably be removed.

**Comment by phoerious** [comment]
I just checked, but I don't see the icons you mention. Can you provide a screenshot?

**Comment by droidmonkey** [comment]
This could be a bug with Qt 5.5

**Comment by markbourne** [comment]
Database unlock dialog (where you wouldn't even expect to need to enter it twice):
![database-unlock](https://user-images.githubusercontent.com/63286927/87850435-2eb80080-c8e8-11ea-9eef-9b9944b9e9d2.png)

Edit entry:
![edit-entry](https://user-images.githubusercontent.com/63286927/87850642-c4a05b00-c8e9-11ea-8b49-eca9e1856523.png)

Password generator:
![password-generator](https://user-images.githubusercontent.com/63286927/87850438-35467800-c8e8-11ea-91bd-dbea82ecce8f.png)

Admittedly not not the biggest issue ever, just slightly confusing when combined with removal of the repeat password field in the "Edit entry" dialog.

I don't know if it's related, but I notice when password fields are first shown, the caps-lock warning icon is also shown at the left but greyed-out; toggling caps lock on and back off removes the icon entirely. Looks like it could be that all the possible icons are always being shown initially when they shouldn't be? (Whether that's a KeePassXC or Qt bug I couldn't say!)

**Comment by phoerious** [comment]
That looks like a Qt bug in 5.5. Please try the AppImage. There are also other known issues in 5.5

**Comment by droidmonkey** [comment]
The bug seems to be in not being able to hide field actions

**Comment by markbourne** [comment]
Using the AppImage, the extra icons are not shown. Looks like Qt 5.14.2 from Help > About > Debug Info.

They're also not shown using the Ubuntu PPA package on a laptop running the newer Linux Mint 19. Looks like Qt 5.9.5. I'll be updating my main PC at some point anyway (the PPA is convenient for updating everything in one place through apt / mintupdate).

Presumably "Qt won't fix it" just means it won't be fixed in the older 5.5 version (which is understandable), since it seems to be OK in newer versions.
